.travis.yml

```diff
@@ -1,16 +1,13 @@
 sudo: false
-language: cpp
-compiler:
-    - gcc
 env:
-    - GXX=g++-4.8 CC=gcc-4.8
+    - GXX=g++-6 CC=gcc-6
 addons:
     apt:
         sources:
             - ubuntu-toolchain-r-test
```

Comment:
Don't forget to change this to `ubuntu-toolchain-r-ppa` since that's what we're using in production